Description: Tweak build system
 The ChangeLog is installed automatically by debhelper, and
 COPYING should be skipped entirely as the information it contains
 are already covered by the Debian copyright file: patch the
 upstream build system to make sure neither is installed.
 .
 Moreover, the ChangeLog must not be regenerated because otherwise
 building the Debian package from a clone of the git repository
 would overwrite the one shipped by upstream.
Forwarded: not-needed
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2017-01-15
Index: beef.git/Makefile.am
===================================================================
--- beef.git.orig/Makefile.am
+++ beef.git/Makefile.am
@@ -7,8 +7,6 @@ SUBDIRS = \
 
 dist_doc_DATA = \
 	AUTHORS.markdown \
-	ChangeLog \
-	COPYING \
 	NEWS.markdown \
 	README.markdown \
 	$(NULL)
@@ -18,19 +16,6 @@ EXTRA_DIST = \
 	$(NULL)
 
 CLEANFILES = \
-	ChangeLog \
 	$(NULL)
 
-ChangeLog:
-	$(AM_V_GEN) \
-	if test -d "$(top_srcdir)/.git"; \
-	then \
-	  GIT_DIR="$(top_srcdir)/.git" \
-	  $(top_srcdir)/missing \
-	    --run git log --color=never --stat \
-	  >$@.tmp \
-	  && mv -f $@.tmp $@ \
-	  || ( $(RM) -f $@.tmp; false ) \
-	fi
-
 .PHONY: ChangeLog
